bugbug/repository.py

```diff
@@ -1552,6 +1553,37 @@ def trigger_pull() -> None:
     trigger_pull()
 
 
+def generate_commit_from_raw_patch(
+    repo_dir: str, base_rev: str, patch: bytes, commit_msg: str
+):
+    # Open the hg repository with minimal extensions to avoid compatibility issues
+    configs = [
+        b"extensions.evolve=!",  # Disable evolve extension
+        b"extensions.mozext=!",  # Disable mozext extension
+    ]
```

Comment:
We are not doing this anywhere else, it's only a problem when running locally with these enabled.
I would remove this and just assume they are not enabled

Reply:
Done here: 95d0879f32a2f3f7a683be4ec9605b86cea6e654